specifyweb/frontend/js_src/lib/components/QueryBuilder/Export.tsx

```diff
@@ -59,6 +66,33 @@ export function QueryExportButtons({
     });
   }
 
+  const [separator] = userPreferences.use(
+    'queryBuilder',
+    'behavior',
+    'exportFileDelimiter'
+  );
+
+  function handleSelectedResults(): string {
```

Comment:
Could you put a comment saying this is only called if the query is not distinct because you can't select rows in distinct queries?